WORKSPACE

```diff
@@ -49,11 +49,10 @@ http_archive(
         "//openxla_patches:cache_urls.diff",
         "//openxla_patches:gpu_race_condition.diff",
         "//openxla_patches:f16_abi_clang.diff",
-        "//openxla_patches:quant_dequant_converter.diff",
```

Comment:
@qihqi @lsy323 it seems that quant_dequant_converter.diff is part of openxla now, could you help confirm?

Reply:
Yes, It's part of openxla now